src/library/basesqltablemodel.h

```diff
@@ -49,7 +49,7 @@ class BaseSqlTableModel : public BaseTrackTableModel {
     TrackId getTrackId(const QModelIndex& index) const override;
     QString getTrackLocation(const QModelIndex& index) const override;
 
-    const QLinkedList<int> getTrackRows(TrackId trackId) const override {
+    const std::list<int> getTrackRows(TrackId trackId) const override {
```

Comment:
Missing `#include <list>` (or whatever is appropriate, see @daschuer's comment)

Reply:
No, this overrides the method from `trackmodel.h` and it's included there.